NitroxServer/Communication/Packets/Processors/PlayerStatsProcessor.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.Serialization.World;
+using NitroxModel.DataStructures.GameLogic;
+
+namespace NitroxServer.Communication.Packets.Processors
+{
+    class PlayerStatsProcessor : AuthenticatedPacketProcessor<PlayerStats>
+    {
+        private readonly World world;
+
+        public PlayerStatsProcessor(World world)
+        {
+            this.world = world;
+        }
+
+        public override void Process(PlayerStats packet, Player player)
+        {
+            world.PlayerData.PlayerStats(player.Name, new PlayerStatsData(packet.Oxygen, packet.MaxOxygen, packet.Health, packet.Food, packet.Water));
```

Comment:
We can put PlayerStatsData as a member variable inside PlayerStats so you can simply pass in packet.data.